File: abc030/src/a.go
package main

import "fmt"

func main() {
    var a, b, c, d int
    fmt.Scan(&a, &b, &c, &d)
    var ans string
    if b*c == d*a {
        ans = "DRAW"
    } else if b*c > d*a {
        ans = "TAKAHASHI"
    } else {
        ans = "AOKI"
    }
    fmt.Println(ans)
}

File: abc030/src/a.cpp
#include <iostream>

using namespace std;

int main() {
    int a, b, c, d;
    cin >> a >> b >> c >> d;
    string ans;
    if (b * c == d * a) {
        ans = "DRAW";
    } else if (b * c > d * a) {
        ans = "TAKAHASHI";
    } else {
        ans = "AOKI";
    }
    cout << ans << endl;
    return 0;
}